src/test/java/placement/model/ShipTests.java

```diff
@@ -0,0 +1,56 @@
+package placement.model;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import placement.model.board.Board;
+import placement.model.ship.Direction;
+import placement.model.ship.Ship;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+public class ShipTests {
+
+    @Test
+    public void givenNewlyCreatedShip_WhenCheckingItsStatus_ThenItIsNotPlaced() {
+        Board board = Board.createDefaultBoard();
```

Comment:
you are using default board in each test.
this could be a part of setUp() method